cloud_sql_proxy/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM scratch
```

Comment:
Just added a second commit to make sure we statically build it :)
$ docker run f013
2016/01/11 23:59:23 No authentication method available! When not running on Google Compute Engine, provide the -credential_file flag.